### DISCUSSION TITLE
What to do with Bootstrap Settings?

### DISCUSSION THREAD

**Discussion by mac-joomla** [templaza/astroid-framework#599] [General]
Hello,
these preferences are looking very interesting, but what are there for and how to use it?

<img width="1012" alt="Bildschirmfoto 2024-03-08 um 14 47 10" src="https://github.com/templaza/astroid-framework/assets/87974913/36ee0699-4456-4945-aa0a-76e92c32f8f5">

I ve been looking to the documentation, but in this case it wasnt helpful for me ;)
https://docs.astroidframe.work/basic-settings/bootstrap-settings

Thank you and best regards

**Comment by Chacapamac** [comment]
It permit to load only what you need in Bootstrap, if not included by Joomla.

**Comment by mac-joomla** [comment]
Thank you, but what will happen, if I need a button and I choose it here? How to get the button?

**Reply by sonvnn** [reply]
If you need an accordion or a carousel in your article or your custom module. You will need it.

**Reply by mac-joomla** [reply]
Thank you, this sounds good, but how exactly I get the button in my article after activating this option in preferences? Is there some code I have to insert?

**Reply by Chacapamac** [reply]
Use the code at —> https://getbootstrap.com/docs/4.0/components/buttons/

**Comment by mac-joomla** [comment]
Thank your for this Link, this makes sense altogether and is a cool feature 👍
